scripts/keploy_record_script.log

```diff
@@ -0,0 +1,30 @@
+
+       ▓██▓▄
```

Comment:
.log files can be removed if not necessary

Reply:
There is a bug that on some platforms causes the extension to fail as it is unable to generate a new log file. This can be fixed in the future so that the log file is generated with the correct permissions. Till then for simplicity, an empty log file is packaged along with the extension